cli/command/init/puppeteer.go

```diff
@@ -0,0 +1,43 @@
+package init
+
+import (
+	// imports embed to load .sauceignore
+	_ "embed"
+
+	"github.com/saucelabs/saucectl/internal/config"
+	"github.com/saucelabs/saucectl/internal/puppeteer"
+)
+
+func configurePuppeteer(cfg *initConfig) interface{} {
+	return puppeteer.Project{
+		TypeDef: config.TypeDef{
+			APIVersion: config.VersionV1Alpha,
+			Kind:       config.KindPuppeteer,
+		},
+		Sauce: config.SauceConfig{
+			Region:      cfg.region,
+			Sauceignore: ".sauceignore",
+			Concurrency: 2, //TODO: Use MIN(AccountLimit, 10)
+		},
+		RootDir: ".",
+		Puppeteer: puppeteer.Puppeteer{
+			Version: cfg.frameworkVersion,
+		},
+		Suites: []puppeteer.Suite{
+			{
+				Name:    "My First Suite", //TODO: Authorize to name you suite
```

Comment:
Same for this one?

Reply:
This `TODO` statement may be remove, as we discussed, it may not be mandatory to ask user to input a suite name.
I'll make it dynamic using platform settings.

Reply:
Updated !